### ISSUE TITLE
[Chore] Set up CD pipelines

### ISSUE THREAD

**Issue by Goose97** [Goose97/google-scraper-ruby#29]
## Why

To save time manually deploying when merging code to develop/main branch

## Acceptance Criteria

- Once a PR is merged into the develop branch, it automatically deployed to the staging environment
- Once a PR is merged into the main branch, it automatically deployed to the production environment
